oceanmesh/edgefx.py

```diff
@@ -122,6 +123,47 @@ def enforce_mesh_gradation(grid, gradation=0.15, crs="EPSG:4326"):
     cell_size = cell_size.flatten("F")
     tmp = gradient_limit([*sz], elen, gradation, 10000, cell_size)
     tmp = np.reshape(tmp, (sz[0], sz[1]), "F")
+    if stereo:
```

Comment:
Is it possible to set the pole in the projection to somewhere over land to avoid this issue?

Reply:
I will try it. In that case this implementation might be overkill